bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/candidate/CandidateObsComparator.java

```diff
@@ -34,7 +34,7 @@ public int compare(Obs o1, Obs o2) {
 			return direction * ((ret != 0) ? ret : -o1.compareTo(o2));
 		}
 		
-		return direction * attr.getComparator().compare(o1, o2);
+		return direction * attr.getComparator(variant.getSchedule().getMiddlePoint()).compare(o1, o2);
```

Comment:
Let's come back to this one.  We need to be sure every time we're using the middle point that there isn't a more accurate option.